app/core/layouts/Header.tsx

```diff
@@ -44,6 +47,13 @@ const Header = ({ existApplicant }: IProps) => {
     }
   };
 
+  const handleClickProfileApplicant = async () => {
+    submit(null, {
+      method: "get",
+      action: `/applicants/${applicantId}`,
+    });
+  };
+
   const handleLogout = async () => {
```

Comment:
Oh! I haven't reviewed this code when it was first added (I mean for example the other `handleClickProfile` function). In general, I think it is better to use proper links instead of onClick actions and functions that depend on react. Links already provide good functionality, you can right-click on them to open in new tab, etc.